easybuild/easyconfigs/x/xtb/xtb-6.3.0-foss-2019a-pre.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'MesonNinja'
+
+name = 'xtb'
+version = '6.3.0'
+
+homepage = 'https://xtb-docs.readthedocs.io'
+description = """Semiempirical Extended Tight-Binding Program Package"""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+toolchainopts = {}  
+
+# Waiting for new release with GCC suport
+# sources = ['v%(version_major_minor)s-pre.1.tar.gz']
+sources = ['master.zip']
+github_account='grimme-lab'
+source_urls = [GITHUB_LOWER_SOURCE]
+
+builddependencies = [
+    ('Meson', '0.51.2', '-Python-3.7.4', ('GCCcore', '8.3.0')),
+    ('Ninja', '1.9.0', '', ('GCCcore', '8.3.0')),
```

Comment:
Why do you use different versions here compared to the Intel build?
https://github.com/easybuilders/easybuild-easyconfigs/blob/c2cd3a9bcd774928a913cccf4c28241a7ec699a6/easybuild/easyconfigs/x/xtb/xtb-6.2.2-hotfix-intel-2017b-Python-3.6.3.eb#L19-L20